AdobeStockAsset/Test/Unit/Model/SaveAssetTest.php

```diff
@@ -8,17 +8,17 @@
 namespace Magento\AdobeStockAsset\Test\Unit\Model;
 
 use Magento\AdobeStockAsset\Model\SaveAsset;
+use Magento\AdobeStockAssetApi\Api\AssetRepositoryInterface;
+use Magento\AdobeStockAssetApi\Api\CategoryRepositoryInterface;
+use Magento\AdobeStockAssetApi\Api\CreatorRepositoryInterface;
 use Magento\AdobeStockAssetApi\Api\Data\AssetInterface;
 use Magento\AdobeStockAssetApi\Api\Data\AssetInterfaceFactory;
 use Magento\AdobeStockAssetApi\Api\Data\CategoryInterface;
 use Magento\AdobeStockAssetApi\Api\Data\CreatorInterface;
-use Magento\AdobeStockAssetApi\Api\AssetRepositoryInterface;
-use Magento\AdobeStockAssetApi\Api\CreatorRepositoryInterface;
-use Magento\AdobeStockAssetApi\Api\CategoryRepositoryInterface;
-use PHPUnit\Framework\MockObject\MockObject;
 use Magento\Framework\Reflection\DataObjectProcessor;
-use PHPUnit\Framework\TestCase;
 use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
 
 /**
  * Test for save asset service.
```

Comment:
Should we add the `SuppressWarnings(PHPMD.CouplingBetweenObjects)` to this test class also?

Reply:
Static checks are not reporting any issues. Why would you like to do so?

Reply:
Yep, you are right. No, reported issue. I checked out to the magento2/phpunit8 branch to test this PR. Everything fine. Thank you for your work @lbajsarowicz